docs/essentials/building-monorepo-projects.md

```diff
@@ -21,6 +21,44 @@ Some advantages of a monorepo approach are:
 
 Semaphore comes with out-of-box support for monorepos.
 
+## Usage examples for change_in
+
+### When a directory changes
+
+```yaml
+blocks:
+  - name: Test WEB server
+    run:
+      when: "change_in('/web-app/')"
+```
+
+### When a file changes
+
+```yaml
+blocks:
+  - name: Unit tests
+    run:
+      when: "change_in('../Gemfile.lock/')"
+```
+
+### When the default branch is main
+
+```yaml
+blocks:
+  - name: Test WEB server
+    run:
+      when: "change_in('/web-app/', {default_branch: 'main'})"
+```
+
+### Exclude the pipeline file
```

Comment:
We could add the following below the title (in the same place in the other document also):

_Note_: If you change pipeline file the Semaphore will consider `change_in` as `true`. The following illustrates how to disable this behaviour.